netboot-oci-artifacts.md

```diff
@@ -1,44 +1,66 @@
-# Packaging Network Boot files using Image manifest
+# Distributing network boot files via image manifest
 
-Netboot files (e.g. PXE) can be packaged as [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
-using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
-and stored in OCI registry.
+Provisioning bootable containers on bare metal systems require OS installer and bootloader files to be distributed along. This specification provides a standard way to encode and describe such payload.
 
-All required files MUST be packaged and stored in the registry together in order to either boot into container,
-or boot into installer which is responsible for deploying bootable image.
-The OCI image manifest MUST NOT contain any associated configuration files required for booting.
+The main storage layout is described in [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
+specification using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+and stored in an OCI registry.
 
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
+
+The key words "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard][c99-unspecified].
+
+An implementation is not compliant if it fails to satisfy one or more of the MUST, MUST NOT, REQUIRED, SHALL, or SHALL NOT requirements for the protocols it implements.  An implementation is compliant if it satisfies all the MUST, MUST NOT, REQUIRED, SHALL, and SHALL NOT requirements for the protocols it implements.
+
+All required files MUST be packaged and stored in the registry together in order to either boot into container, or boot into installer which is responsible for deploying bootable image. The OCI image manifest MUST NOT contain any associated configuration files required for booting.
```

Comment:
I don't think this belongs here, maybe place it back where it was previously. 
This section defines generic conventions around keywords usage.

Reply:
I am trying to put the conventions before first use. Do you really want them at the end of the document? I can maybe create a link there at the beginning.

Reply:
no, I meant partivularly last paragraph:
All required files MUST be packaged and stored in the registry together in order to either boot into container, or boot into installer which is responsible for deploying bootable image. The OCI image manifest MUST NOT contain any associated configuration files required for booting.

I think it you could keep it in `# Distributing network boot files via image manifest`